app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 80px;
+  background-color: #000;
```

Comment:
Let's use appsilon navy color as a tooltip background (#15354a) instead of normal black

Reply:
Awesome idea, changed